src/frontend/js/components/Tabs/_styles.scss

```diff
@@ -0,0 +1,106 @@
+// .tabs{
+//   $background-color: r-theme-val(dashboard, background-color);
+
+//   &__header{
+//     display:flex;
+//     border-bottom : 2px solid gray;
+//     padding: 0 rem-calc(16px) 0 rem-calc(16px);
+
+//     &__tab {
+//       padding: rem-calc(4px) rem-calc(4px) 0 rem-calc(4px);
+//       border-radius:rem-calc(8px) rem-calc(8px) 0 0;
+//       border: 2px solid gray;
+//       border-bottom : 0;
+//       flex:0 0 auto;
+
+//       margin-right: rem-calc(8px);
+//       &:last-child{
+//         margin-right: 0;
+//       }
+
+//       &--active {
+//         border-bottom: 2px solid $background-color;
+//         margin-bottom: -2px;
+//       }
+//     }
+
+//     margin-bottom: rem-calc(16px);
+//   }
+// }
+
+$button-tab-scheme: map-get(
+  map-get(map-get(map-get($themes, 'default'), 'components'), 'button'),
+  'tab'
+);
+$default-border-radius: map-get($button-tab-scheme, 'border-radius');
+$default-color: map-get($button-tab-scheme, 'default-color');
+$hover-color: map-get($button-tab-scheme, 'hover-color');
+$focus-color: map-get($button-tab-scheme, 'focus-color');
+$selected-color: map-get($button-tab-scheme, 'selected-color');
+$disabled-color: map-get($button-tab-scheme, 'disabled-color');
+
+$tabs-scheme: map-get(map-get(map-get($themes, 'default'), 'components'), 'tabs');
+.tabs {
+  &__header {
+    display: flex;
+    border-bottom: 2px solid map-get($tabs-scheme, 'border-bottom-color');
+
+    &__tab {
+      margin-bottom: rem-calc(-2px);
+      flex: 0 0 auto;
+
+      margin-right: rem-calc(16px);
+      &:last-child {
+        margin-right: 0;
+      }
+    }
+  }
+}
+
+.c__button {
+  $default-color: map-get($button-tab-scheme, 'default-color');
+
+  &--tab {
+    // avoid movement when hovering/selecting a tab.
+    border: 1px solid transparent;
+    border-bottom-width: 2px;
+
+    padding-left: 0;
+    padding-right: 0;
+    color: $default-color;
+    background-color: transparent;
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 0;
```

Comment:
I do not see where you are applying `border-radius`.

Reply:
Cunningham's button already have border radius all around.
I removed the bottom one to fit with the tab's underline.

Reply:
I wonder if we could not use button text variant ?

Reply:
It's basicaly what this is, the `--tab` button is a override of cunningham's text button.
It's everything the text button is, with ajustement to be a tab button in tabs design.

I can move the class outside of `.c__button` and move the variable outside of `components.button` but at the end, it'll be a cunningham button with some dedicated css.
It's look like a button variant.

Reply:
There are text variant for buttons in Cunningham. We could use them directly. That's what I meaned.

https://openfun.github.io/cunningham/storybook/?path=/story/components-button--primary-text